src/libsync/configfile.cpp

```diff
@@ -863,7 +864,16 @@ void ConfigFile::setNewBigFolderSizeLimit(bool isChecked, qint64 mbytes)
 
 bool ConfigFile::confirmExternalStorage() const
 {
-    return getValue(confirmExternalStorageC, QString(), true).toBool();
+    const auto fallback = getValue(confirmExternalStorageC, QString(), true);
+    const auto value = getPolicySetting(QLatin1String(confirmExternalStorageC), fallback);
```

Comment:
You can return `getPolicySetting(QLatin1String(useNewBigFolderSizeLimitC), fallback).toBool();` immediately here. The `value` is not used anywhere else anyway. Just an extra line of code that can be removed without suffering readability too much.